File: packages/libbird/android/src/main/java/dev/flutterbird/ladybird/ImageDecoderService.java
package dev.flutterbird.ladybird;

import android.os.Message;

public final class ImageDecoderService extends LadybirdServiceBase {
    public ImageDecoderService() {
        super("ImageDecoderService");
    }

    @Override
    boolean handleServiceSpecificMessage(Message msg) {
        return false;
    }

    static {
        System.loadLibrary("imagedecoderservice");
    }
}

File: packages/libbird/android/src/main/kotlin/org/serenityos/ladybird/ImageDecoderService.kt
package org.serenityos.ladybird

import android.os.Message

class ImageDecoderService : LadybirdServiceBase("ImageDecoderService") {
  override fun handleServiceSpecificMessage(msg: Message): Boolean = false

  companion object {
    init {
      System.loadLibrary("imagedecoderservice")
    }
  }
}
